src/js/logout/main.js

```diff
@@ -1,5 +1,7 @@
+'use strict';
+
 import SimpleSAMLLogout from './logout.js';
 
-$(document).ready(function () {
-    new SimpleSAMLLogout($('body').attr('id'));
-});
\ No newline at end of file
+ready(function () {
+    new SimpleSAMLLogout(document.body.id);
+});
```

Comment:
I'm experiencing some kind of timing-issue that causes a "partial logout" instead of a "global logout" message even though all SPs have been logged out.. When I revert this to the jQuery-equivalent things start working again..
@thijskh Any ideas here?

Reply:
It could be an issue with the extra level of indirection here. Before, jquery was loaded, and then when we reached this code we would use it to register the handler. Now it is kind of inverted. We need to load `ready.js` first so that `ready()` is declared at this point, but the handler is registered back there in `ready.js`. So if the state changes *before* we reach this code, it would run `handleState()`, check the `window.readyHandlers` list, and it will be empty, so it won't do anything else and our code will never be run.

This is just speculation, I'm not sure that's what's happening here, but it could be...

Reply:
OK, so I replaced `ready.js` with event handlers all over the place and now the Office webviewer throws syntax errors again.. :(

Reply:
The issue existed in `master` already and IframeLogout is broken in 1.19 (at least for the newui), so I don't see this as a blocking issue anymore.